tests/Tests/500__jupyterhub/test-jupyterlab-time.robot

```diff
@@ -0,0 +1,83 @@
+*** Settings ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             ../../../../libs/Helpers.py
+Library             DateTime
+Library             OpenShiftCLI
+Library             DebugLibrary
+
+Suite Setup         Load Spawner Page
+Suite Teardown      End Web Test
+
+
+*** Variables ***
+@{LIST_OF_IMAGES}       s2i-minimal-notebook    s2i-generic-data-science-notebook
+...                     pytorch                 tensorflow    minimal-gpu
+
+${LIMIT_TIME} =    40
+
+*** Test Cases ***
+Average Time For Spawning
```

Comment:
suggestion for a more self-explanatory title: `Verify Average Spawn Time Is Less Than 40 Seconds`

Reply:
Done